pb/nodemessage.proto

```diff
@@ -266,3 +305,27 @@ message SignatureChainObjectionSigned {
   bytes message = 1;
   bytes signature = 2;
 }
+
+// sync txn pool
+message RplTxnPoolHash {
```

Comment:
Better be consistent with abbreviation and use `reply` instead of `rpl`

Reply:
fixed